processor/redactionprocessor/internal/db/config.go

```diff
@@ -0,0 +1,97 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package db
+
+import "go.opentelemetry.io/otel/semconv/v1.32.0/dbconv"
+
+type DBSanitizerConfig struct {
```

Comment:
Do we need to expose all of this configuration in the first version? I would rather keep it very simple for the first version and include these only as users request them